config.json

```diff
@@ -729,6 +729,14 @@
         "nugetId": "Xamarin.AndroidX.SavedState",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.security",
+        "artifactId": "security-crypto",
+        "version": "1.0.0-rc03",
+        "nugetVersion": "1.0.0-rc03",
+        "nugetId": "Xamarin.AndroidX.Security.SecurityCrypto",
```

Comment:
Might want to consider removing a redundant `Security`:
`Xamarin.AndroidX.Security.Crypto`

Precedent:
`androidx.slice.slice-builders` is `Xamarin.AndroidX.Slice.Builders`